Ariel/Ariel/Auxiliary/DegreesConverter.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  DegreesConverter.swift
+//  Ariel
+//
+//  Created by Alina Zaitseva on 5/17/18.
+//  Copyright © 2018 Alina Zaitseva. All rights reserved.
+//
+
+import Foundation
+
+class DegreesConverter {
```

Comment:
Suggestions:
- class -> struct
- two properties 'degreesFahrenheit' and `degreesCelsius`
- two inits: `withFahrenheit` and `withCelsius`; each other sets appropriate property and calculates another one
- two more properties `stringPresentationFahrenheit` and `stringPresentationCelsius`